cmd/debos/debos.go

```diff
@@ -272,6 +273,7 @@ func main() {
 			options.CPUs = 2
 		}
 		m.SetNumCPUs(options.CPUs)
+		m.SetSectorSize(r.SectorSize)
```

Comment:
you need to mention a dependency on the PR in fakemachine && also update go.mod to your WIP branch

Reply:
Ah yes so obvious Thanks

Reply:
should be ok now, but I'm not familiar with go, so let see :)